.vscode/settings.json

```diff
@@ -1,3 +1,3 @@
 {
-  "cSpell.words": ["lukso"]
+  "cSpell.words": ["lukso", "lyxe"]
```

Comment:
Let's please not commit personal configuration files

Can you add .vscode folder to gitignore?

Reply:
Sure 👍 Didn't know I was doing it 😅